pkg/common/consumer/consumer_handler.go

```diff
@@ -52,12 +53,22 @@ func WithSaramaConsumerLifecycleListener(listener SaramaConsumerLifecycleListene
 	}
 }
 
+// WithTimeout configures the request timeout. Default is set to 60s.
+func WithTimeout(timeout time.Duration) SaramaConsumerHandlerOption {
```

Comment:
where exactly is that called / used ?

Reply:
I believe that's up for us to use it when composing the factory function `NewConsumerHandler` to override the default 60s. Can happen on a later PR though.